agent/agents/postgres/pgstatmonitor/pgstatmonitor_test.go

```diff
@@ -181,7 +181,7 @@ func TestPGStatMonitorSchema(t *testing.T) {
 	var mPlansCallsCnt, mPlansTimeCnt float32
 	pgsmVersion, _, err := getPGMonitorVersion(db.Querier)
 	assert.NoError(t, err)
-	switch pgsmVersion {
+	switch pgsmVersion { //nolint:exhaustive //TODO:@Nurlan please check if this was good warning and we should add more cases
```

Comment:
@BupycHuk and that too

Reply:
Looks ok for me.
@JiriCtvrtka PTAL

Reply:
Since PMM now already supports also PGSM 1.1 and 2.0 I added these cases.